models/prices/prices_tokens.sql

```diff
@@ -1756,5 +1757,56 @@ VALUES
     ("usdt-tether", "solana", "USDT", "Es9vMFrzaCERmJfrF4H2FYD4KCoNkY11McCe8BenwNYB", 6),
     ("drip-drip-network", "bnb", "DRIP", "0x20f663CEa80FaCE82ACDFA3aAE6862d246cE0333", 18),
     ("fur-furio", "bnb", "$FUR", "0x48378891d6E459ca9a56B88b406E8F4eAB2e39bF", 18),
-    ("pstn-piston", "bnb", "PSTN", "0xBfACD29427fF376FF3BC22dfFB29866277cA5Fb4", 18)
+    ("pstn-piston", "bnb", "PSTN", "0xBfACD29427fF376FF3BC22dfFB29866277cA5Fb4", 18),
+
+    ("weth-weth", "fantom", "WETH", "0x74b23882a30290451A17c44f4F05243b6b58C76d", 18), 
+    ("wbtc-wrapped-bitcoin", "fantom", "WBTC", "0x321162Cd933E2Be498Cd2267a90534A804051b11", 8),
+    ("ftm-fantom", "fantom", "WFTM", "0x21be370D5312f44cB42ce377BC9b8a0cEF1A4C83", 18),
+    ("aave-new", "fantom", "AAVE", "0x6a07A792ab2965C72a5B8088d3a069A7aC3a993B", 18), 
+    ("avax-avalanche", "fantom", "AVAX", "0x511D35c52a3C244E7b8bd92c0C297755FbD89212", 18), 
+    ("crv-curve-dao-token", "fantom", "CRV", "0x1E4F97b9f9F913c46F1632781732927B9019C68b", 18),
```

Comment:
`ERROR:root:WARN: crv-curve-dao-token Provided contract address: 0x1E4F97b9f9F913c46F1632781732927B9019C68b not in CoinPaprika contracts list ['0xd533a949740bb3306d119cc777fa900ba034cd52']. (Not uncommon! share block explorer link to confirm contract)`
- API: https://api.coinpaprika.com/v1/coins/crv-curve-dao-token
- SCAN: https://ftmscan.com/token/0x1E4F97b9f9F913c46F1632781732927B9019C68b